.buildkite/pipelines/security_solution/api_integration.yml

```diff
@@ -0,0 +1,21 @@
+steps:
+  - label: Running exception_workflows:runner:serverless
```

Comment:
These should have a queue and a timeout defined

Reply:
I realized this a bit late, and changed it before I saw that comment. Thanks a lot @jbudz